kubernetes/cray-istio/values.yaml

```diff
@@ -21,7 +21,7 @@
 # Declare variables to be passed into your templates.
 kubectl:
   image:
-    repository: dtr.dev.cray.com/loftsman/loftsman
+    repository: arti.dev.cray.com/csm-docker-stable-local/loftsman
```

Comment:
The appropriate image to use for `kubectl` is `artifactory.algol60.net/csm-docker/stable/docker-kubectl:1.19.15`.